modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase

```diff
@@ -198,12 +198,59 @@ definition {
 	}
 
 	@description = "This is a test for LPS-123538. As Content Creator, I want to define reusable translations when creating WEM Structures with fieldset"
-	@ignore = "true"
 	@priority = "5"
 	test EditFieldset {
 		property portal.acceptance = "false";
 
-		// TODO LPS-123538 EditFieldset pending implementation
+		var fieldsetLabel = "My New Fieldset";
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+
+		AppBuilderAdmin.switchTab(tabName = "Document Types");
+
+		DMDocumentType.addWithoutSave(
+			dmDocumentTypeFieldNames = "Numeric",
+			dmDocumentTypeName = "Document Type Name");
```

Comment:
Hey @Odrakir1 , this test should be made under Web Content / Stuctures and not in Documents and Media / Document Type.